docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.3'
+
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    command: --default-authentication-plugin=mysql_native_password
+    env_file:
+      - ./mysql/.env
+    ports:
+      - 3306:3306
+    expose:
+       - 3306
+
+
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+
+    command: bash -c "python database/admin.py && python app/handle_msg.py"
+    volumes:
+      - .:/code
```

Comment:
so it's syncing current dir on the host with the `code` directory in the container root directory. I'm pretty much sure it's not what you need